project/{{ cookiecutter.__folder_name }}/devops/tasks/swarm/task_swarm.yml

```diff
@@ -11,7 +11,31 @@
         advertise_addr: "{{ swarm.advertise_addr }}"
       register: swarm_info
 
+- name: "Docker Swarm: Set public network"
+  become: true
+  become_user: "{{ users.default.name }}"
+  block:
+    - name: "Docker Swarm: Check for public network"
+      community.docker.docker_network_info:
+        name: "{{ cookiecutter.__devops_swarm_public_network }}"
+      register: docker_swarm_net_public
+
+    - name: "Docker Swarm: Create public network"
+      community.docker.docker_network:
+        name: "{{ cookiecutter.__devops_swarm_public_network }}"
```

Comment:
There is some issue with the cookiecutter variables here. They were not replaced by the correct variable.

Reply:
Ah, it's because `"devops/tasks"` is in cookiecutter.json `copy_without_render`

Reply:
Good catch, I will fix this now

Reply:
There is something else with the `projects_folders` variable definition in `devops/inventory/group_vars/all/projects.yml` I see in the template that the definition is correct, it has a leading dash (-).

But when the project structure is created, the dash is missing:

TEmplate:

```
project_folders:
  # Path to store {{ cookiecutter.hostname }} postgres data
  - path: "{{ lookup('ansible.builtin.env', 'DEPLOY_FOLDER', default='/srv/plone/data') }}"
    owner: 999
    group: 999
    mode: "0750"
```

Generated structure:

```
project_folders:
  # Path to store {{ cookiecutter.hostname }} postgres data
    path: "{{ lookup('ansible.builtin.env', 'DEPLOY_FOLDER', default='/srv/plone/data') }}"
    owner: 999
    group: 999
    mode: "0750"
```
It may also be related with the `copy_without_render` thing because this folder is also there.

Reply:
We also need a solution for passing the env vars when doing the stack deploy, otherwise the `DEPLOY_FOLDER` is not correctly resolved when deploying. 

This is what we talk through discord :/

Reply:
I fixed the copy_without_render and I plan to merge this asap.
Then I will open a new PR refactoring the ansible part to address the passing of variables to the stack during deployment